### ISSUE TITLE
TV System support

### ISSUE THREAD

**Issue by oyvindln** [oyvindln/ld-decode#18]
Tape formats (vhs-decode part):
- [x] NTSC
- [x] PAL (normal 625 line)
- [ ] SECAM-L
- [ ] MESECAM (VHS specific) - some preliminary support
- [X] PAL-M
- [ ] PAL-N (Not sure if there is any practical difference from normal PAL on tape)

CVBS and ld-chroma-decoder part:
- [x] NTSC
- [x] PAL (normal 4.43 625 line)
- [ ] PAL-60
- [ ] PAL-M (think this would work?)
- [ ] PAL-N
- [ ] SECAM
- [ ] NTSC 4.43

**Comment by cher-nov** [comment]
Just FYI: there's a similar project supporting SECAM decoding from VHS.
https://github.com/grishka/AVDecoder - _partially in Russian, use online translator_

**Comment by harrypm** [comment]
> Just FYI: there's a similar project supporting SECAM decoding from VHS.
> https://github.com/grishka/AVDecoder - _partially in Russian, use online translator_

Yup he's in the community, there's still a little bit of work to be done still for SECAM but it's now in the tools, next thing will be some metadata updates for it and dedicated profiles for the more professional formats as more samples come in now Etinne is all settled in with with the MISRC V2.5.
